src/xdocs/google_style.xml

```diff
@@ -2437,6 +2437,31 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Overrides" id="Google_Overrides">
+        <p>
+          To change the default severity level, use the following custom property variable:
+          <br/>
+          <br/>
+          <code>org.checkstyle.google.severity=${SeverityLevel}</code>
+          <br/>
+          <br/>
+          where <code>SeverityLevel</code> can be any one of them:
+          <code>ignore</code>, <code>info</code>, <code>warning</code>, <code>error</code>.
+          Default is <code>Warning</code>.
```

Comment:
better to remove this block

Reply:
If there is a concern we need to guide users on what a severity is or how it should be properly used, we should point them to https://checkstyle.org/config.html#Severity .

@romani Please clarify what she be removed. It seems to me we need to warn users what the default is, but I have not read this documentation yet.

Reply:
I have removed that block for now. If we decide to modify it, then I will bring that block back

Reply:
Yes, link to severity is what we need. No text.

Reply:
I am good with latest text or new section